### DISCUSSION TITLE
Delay in calling 'onDidCloseTextDocument' in vscode 1.86.2

### DISCUSSION THREAD

**Discussion by sajithpt** [microsoft/vscode-discussions#1104] [Extension Development QnA]
Is there a change in vscode 1.86.2 that result in delay in calling 'onDidCloseTextDocument' event when an 'Untitled' file is closed?
I understand that 'onDidCloseTextDocument' is not guaranteed to get called immediately on closing a file. 
But till vscode 1.84.2, this gets called without any noticeable delay.
But in 1.86.2, 'onDidCloseTextDocument' gets called almost 3 minutes after an 'Untitled' file is closed. Same is the case with 1.87.2. 
Is there a change in 1.86.2 with which this delay is expected or is this an issue that will be fixed?

**Comment by isidorn** [comment]
This sounds like a bug. If you can still reproduce it with the latest VS Code version I suggest you open an issue here https://github.com/microsoft/vscode/issues

Thank you!
